### ISSUE TITLE
NOTIFICATION_CONFIRM_SUBMAIL has a %s, but is not called with a variable

### ISSUE THREAD

**Issue by onli** [s9y/Serendipity#770]
Investigation issue.

The comment confirmation message is somewhat broken, as NOTIFICATION_CONFIRM_SUBMAIL has a %s that is not filled. The german text for that variable has no %s. Is that a sign that the language constant files are in chaos or is it a single and simple error?

**Comment by onli** [comment]
https://github.com/s9y/Serendipity/commit/0cf227f3352c999786de7daa055e53c9d759dca6 did introduce it. A sensible change probably for the alternative comment subscription system. Could be readded later (with the outer code to fill it), but let's keep it simple for now.

**Comment by stephanbrunker** [comment]
Sorry for introducing that bug. I didn't intend to mixup the re-sorting of the language files with the new feature. Unfortunately, I was already several steps ahead in my developing HEAD and missed that I already made these changes when I committed (kind of cherry-picked) the language file makeover. Originally, the strings were without the %s and my action also caused the vanishing of some constants like NOTIFICATION_CONFIRM_SUBMAIL_FAIL. It was never intended that my HEAD would never committed, because then everything should be consistent again.

It is something of a vicious circle because I did the makeover of the language files (which was sensible) only because the later changes made everything even more messed up.
